workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -755,7 +877,11 @@ def scale(self):
                 if len(additional_hosts) < self.max_additional_hosts:
                     Logger.info('There are %s/%s additional child pipelines. Scaling up will be performed.' %
                                 (len(additional_hosts), self.max_additional_hosts))
-                    self.scale_up()
+                    resource = self.grid_engine.get_resource_demand(expired_jobs)
```

Comment:
I'm concerned that we use `expired_jobs` here and not `pending_jobs`. Just imagine that we have only 1 expired job (it was in queue for 30 seconds) and 1000 jobs that were submitted within last 30 seconds. We'll scale up small instance rather than a big one. Let's change this.